Lib/posixpath.py

```diff
@@ -36,7 +36,8 @@
            "samefile","sameopenfile","samestat",
            "curdir","pardir","sep","pathsep","defpath","altsep","extsep",
            "devnull","realpath","supports_unicode_filenames","relpath",
-           "commonpath", "isjunction","isdevdrive","ALLOW_MISSING"]
+           "commonpath", "isjunction","isdevdrive",
+           "ALL_BUT_LAST", "ALLOW_MISSING"]
```

Comment:
```suggestion
           "commonpath","isjunction","isdevdrive",
           "ALL_BUT_LAST","ALLOW_MISSING"]
```

Should we keep leaving the spaces out here? Or add them all? I don't particularly care, just noticed it was inconsistent.